browser/react/components/Beth.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { joinChatRoom, leaveChatRoom } from '../../webRTC/client.js';
+import Teleporter from './Teleporter';
+
+export default class Beth extends React.Component {
+
+  componentDidMount () {
+    joinChatRoom('beth');
+  }
+
+  componentWillUnmount () {
+    leaveChatRoom('beth');
+  }
+  render () {
+    return (
+      <a-entity id="room" position="0 0 0">
```

Comment:
we could replace this for everyone's rooms with my Room react component after I merge.